src/superphot_plus/model/classifier.py

```diff
@@ -128,14 +127,9 @@ def train_and_validate(
         Returns
         -------
         tuple
-            A tuple containing the best validation loss and accuracy.
+            A tuple containing arrays of metrics for each epoch
+            (training accuracies and losses, validation accuracies and losses).
         """
-        random.seed(SEED)
```

Comment:
@kdesoto-astro I was wondering, should we should keep this code that sets the seed from this point on?

Reply:
We should probably move this to an rng_seed input arg like in the fitting case, and simply set to a random seed if that arg is None. That means removing the SEED default arg from the constants file